packages/db-ra-data-rest/babel.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  presets: [
```

Comment:
Answering to [your questions](https://github.com/platformatic/platformatic/pull/243#issue-1428358242), my take on babel is that if we don't need it, we shouldn't have it. So, not sure why we should have this if this library is pure standard js not using "next" features. "If it isn't there, it can't break", but maybe here I am missing something on the reason why we should have it

Reply:
ouch this is a leftover of when I was using jest which needed babel